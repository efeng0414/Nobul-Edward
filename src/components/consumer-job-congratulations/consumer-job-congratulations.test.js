import React from "react";
import { BrowserRouter } from "react-router-dom";
import ConsumerJobCongratulations from "./index";
describe("ConsumerJobCongratulations", () => {
  const componentProps = {
    intl: global.getIntl(),
    visible: true,
    onCancel: () => {}
  };
  const componentJSX = <ConsumerJobCongratulations {...componentProps} />;
  const contextObj = {
    childContextTypes: {
      router: () => void 0
    },
    context: {
      intl: componentProps.intl,
      router: {
        history: new BrowserRouter().history || "",
        route: {
          location: {}
        }
      }
    }
  };
  it("should match existing snapshot", () => {
    const wrapper = Enzyme.shallow(componentJSX, contextObj);
    expect(wrapper).toMatchSnapshot();
  });
});
