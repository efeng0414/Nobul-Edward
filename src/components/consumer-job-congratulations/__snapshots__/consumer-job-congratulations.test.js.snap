// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsumerJobCongratulations should match existing snapshot 1`] = `
<Modal
  cancelButtonDisabled={false}
  confirmLoading={false}
  footer={null}
  maskTransitionName="fade"
  okButtonDisabled={false}
  okType="primary"
  onCancel={[Function]}
  prefixCls="ant-modal"
  transitionName="zoom"
  visible={true}
  width={520}
>
  <div
    className="congratulations"
  >
    <div
      className="congratulations-image"
    >
      <img
        alt="CONGRATULATIONS"
        src="file-mock"
      />
    </div>
    <div
      className="congratulations-text"
    >
      <p
        className="congratulations-text-title"
      >
        CONGRATULATIONS
      </p>
      <p
        className="congratulations-text-content"
      >
        You have just created a posting.
      </p>
      <p
        className="congratulations-text-content"
      >
        An agent will soon be in touch with you.
      </p>
    </div>
  </div>
</Modal>
`;
